hydromt_wflow/wflow.py

```diff
@@ -934,12 +937,17 @@ def setup_lulcmaps(
             da=da,
             ds_like=self.grid,
             df=df_map,
-            params=lulc_vars,
+            params=list(lulc_vars.keys()),
             logger=self.logger,
         )
         rmdict = {k: v for k, v in self._MAPS.items() if k in ds_lulc_maps.data_vars}
         self.set_grid(ds_lulc_maps.rename(rmdict))
 
+        # Add entries to the config
```

Comment:
This is a nice addition and we should do this for all the setup methods but let's so that in another PR. I also wonder if then we should start simplifying and removing from our template config. For now not but let's do this when we implement this for all setup methods

Reply:
Yeah I agree that that would be much nicer and much more robust. Indeed something to fix in another PR, but decided to already make a small step in this PR.